rust/src/modbus/log.rs

```diff
@@ -27,7 +27,6 @@ pub extern "C" fn rs_modbus_to_json(tx: &mut ModbusTransaction, js: &mut JsonBui
 
```

Comment:
this commit does a lot more than the commit message indicates

Reply:
Indeed, so, any ideas about the commit segmentation ? 
One big commit ? One commit per protocol ? (but if so, the first commit introducing the generic changes should also be applied on some protocol, otherwise, commit-check will fail because of unused functions)

Should I split this PR ?